internal/config/config.go

```diff
@@ -302,6 +302,13 @@ type AttestationConfig struct {
 	QEMUVTPM *QEMUVTPM `yaml:"qemuVTPM,omitempty" validate:"omitempty,dive"`
 }
 
+// ConfigureToMarshalNumericalValuesInsteadOfLatest configures the AttestationConfig to marshal numerical values instead of the latest version.
+func (c *AttestationConfig) ConfigureToMarshalNumericalValuesInsteadOfLatest() {
```

Comment:
```suggestion
// SetWantLatestFalse sets the WantLatest field to false for all versions in the AttestationConfig to marshal numerical values instead of the latest version.
func (c *AttestationConfig) SetWantLatestFalse() {
```
Since this actually changes the config, the function should be named to reflect that change.

Thoughts about moving this functionality directly into `GetAttestationConfig`?
That way callers will always receive a config that is marshalled as expected without having to perform extra steps.